packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_17_Mining_Rewards.json

```diff
@@ -0,0 +1,199 @@
+{
+  "week": 17,
+  "fromBlock": 11271761,
+  "toBlock": 11317321,
+  "poolAddress": "0xcce41676a4624f4a1e33a787a59d6bf96e5067bc",
+  "blocksPerSnapshot": 256,
+  "umaPerWeek": 14690,
+  "shareHolderPayout": {
+    "0x07d28c64e033dd7f172764a389663580d32e4388": "18.570703917994437582",
+    "0x0a22ef68c6e9f47c3c6f99aebf79e8d4482dcea5": "74.804804153846713182",
+    "0x0a235473a00e1bfe128fd40cb1c0b052cdeabe41": "0.885880773685103175",
+    "0x0fc4b69958cb2fa320a96d54168b89953a953fbf": "373.726655076937164176",
+    "0x16c19dcc764767909059b29cccd0448f4c8819df": "0.207213117365569373",
+    "0x16f037a3ddf53da1b047a926e1833219f0a8e1fc": "128.622233457441955",
+    "0x181c2d506b4aad984e3a98f09ac8eaf895671599": "131.168408056614929086",
+    "0x186cf5714316f47bc59e30a850615a3f938d7d79": "144.324305676187406122",
+    "0x18db8b99c1d6e439fa44fd87bfb1109e345e98da": "14.938398734578934199",
+    "0x1dd50763ebf30966922c428177b090a90a634de9": "9.156741383956258899",
+    "0x1e4bda16734c1f9e86a66e310fcf8bd3c4f9c0f0": "359.648163704539283323",
+    "0x1ee10aeaebc6336b813daf9067248fa90de2b589": "46.041856405414321816",
+    "0x1f295ff3ea09d8eac8a39e3617bd4646d4df7c9a": "120.171047337870375806",
+    "0x230e7f448f6ebc7c4433cce03ee92eeb81681a2c": "198.804194674639862899",
+    "0x25125e438b7ae0f9ae8511d83abb0f4574217c7a": "212.04219245372769194",
+    "0x25ae7b45d8646580dfcae403d29164729eb8642f": "2452.530513816210079689",
```

Comment:
Another big one here just want to make sure you sanity checked

Reply:
yup, I did. this is the biggest sponsor:
![image](https://user-images.githubusercontent.com/12886084/100121004-4a787800-2e81-11eb-8c31-504689af4e12.png)